sdk/core/azure-core/src/main/java/com/azure/core/implementation/logging/DefaultLoggerBase.java

```diff
@@ -0,0 +1,841 @@
+    @Override
+    public void trace(final String msg, final Throwable t) {
+
+    }
+
+    /**
+     * Similar to {@link #isTraceEnabled()} method except that the
+     * marker data is also taken into account.
+     *
+     * @param marker The marker data to take into consideration
+     * @return True if this Logger is enabled for the TRACE level,
+     * false otherwise.
+     * @since 1.4
+     */
+    @Override
+    public boolean isTraceEnabled(final Marker marker) {
+        return false;
+    }
+
+    /**
+     * Log a message with the specific Marker at the TRACE level.
+     *
+     * @param marker the marker data specific to this log statement
+     * @param msg the message string to be logged
+     * @since 1.4
+     */
+    @Override
+    public void trace(final Marker marker, final String msg) {
+
+    }
+
+    /**
+     * This method is similar to {@link #trace(String, Object)} method except that the
+     * marker data is also taken into consideration.
+     *
+     * @param marker the marker data specific to this log statement
+     * @param format the format string
+     * @param arg the argument
+     * @since 1.4
+     */
+    @Override
+    public void trace(final Marker marker, final String format, final Object arg) {
+
+    }
+
+    /**
+     * This method is similar to {@link #trace(String, Object, Object)}
+     * method except that the marker data is also taken into
+     * consideration.
+     *
+     * @param marker the marker data specific to this log statement
+     * @param format the format string
+     * @param arg1 the first argument
+     * @param arg2 the second argument
+     * @since 1.4
+     */
+    @Override
+    public void trace(final Marker marker, final String format, final Object arg1, final Object arg2) {
+
+    }
+
+    /**
+     * This method is similar to {@link #trace(String, Object...)}
+     * method except that the marker data is also taken into
+     * consideration.
+     *
+     * @param marker the marker data specific to this log statement
+     * @param format the format string
+     * @param argArray an array of arguments
+     * @since 1.4
+     */
+    @Override
+    public void trace(final Marker marker, final String format, final Object... argArray) {
+
+    }
+
+    /**
+     * This method is similar to {@link #trace(String, Throwable)} method except that the
+     * marker data is also taken into consideration.
+     *
+     * @param marker the marker data specific to this log statement
+     * @param msg the message accompanying the exception
+     * @param t the exception (throwable) to log
+     * @since 1.4
+     */
+    @Override
+    public void trace(final Marker marker, final String msg, final Throwable t) {
+
+    }
+
+    /**
+     * Is the logger instance enabled for the DEBUG level?
+     *
+     * @return True if this Logger is enabled for the DEBUG level,
+     * false otherwise.
+     */
+    @Override
+    public boolean isDebugEnabled() {
+        return false;
```

Comment:
* Since default logger base is package private abstract class, we always need to have it in DefaultLogger. However API is not that useful for current structure. 
* It is not always the case to read env. We might need to introduce reading from file and etc in future. So it better to have it in subclass instead of base class.
I think to leave it as it is. And we can easily implement this if there is a need in future.